react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+class Stopwatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isClicked: false,
+      seconds: 0
+    };
+    this.handleClick = this.handleClick.bind(this);
+    this.resetTimer = this.resetTimer.bind(this);
+    this.intervalID = null;
+  }
+
+  handleClick() {
+    this.setState({ isClicked: !this.state.isClicked });
```

Comment:
Having this line here can be misleading, it should be at the end of the method. 

It is misleading because the updated value from this `setState` call is not what is being checked in the `if` statement. Remember that `setState` is async so the `if` statement will be checking what the value in state was before the `setState` completes. By moving this line to the end of the method won't change how it is currently working but it will make your code more understandable.